server/server.js

```diff
@@ -2,20 +2,36 @@ const express = require('express');
 const session = require('express-session');
 const bodyParser = require('body-parser');
 const passport = require('passport');
-const LocalStrategy = require('passport-local');
-const bcrypt = require('bcryptjs');
 const redis = require('connect-redis')(session);
 const flash = require('connect-flash');
+const kanban = require('./routes/kanban')
 
-const PORT = process.env.PORT || 8080;
+const PORT = process.env.EXPRESS_CONTAINER_PORT;
 const ENV = process.env.NODE_ENV || 'development';
-
+const SESSION_SECRET = process.env.SESSION_SECRET || 'keyboard cat';
+const REDIS = process.env.REDIS_HOST_PORT;
+const REDIS_URL = process.env.REDIS_URL;
 const app = express();
 
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 app.use(express.static('public'));
+app.use(flash());
```

Comment:
I don't believe `flash` is being used in this project as that's a way to show an error page on a server-side rendered application.

Reply:
Make sure the package is removed from your `package.json` as well